packages/sveltekit/src/vite/types.ts

```diff
@@ -0,0 +1,220 @@
+    ignore?: string | Array<string>;
+
+    /**
+     * A glob or an array of globs that specifies the build artifacts that should be deleted after the artifact
+     * upload to Sentry has been completed.
+     *
+     * @default [] - By default no files are deleted.
+     *
+     * The globbing patterns follow the implementation of the glob package. (https://www.npmjs.com/package/glob)
+     */
+    filesToDeleteAfterUpload?: string | Array<string>;
+  };
+
+  /**
+   * Options related to managing the Sentry releases for a build.
+   *
+   * Note: Managing releases is optional and not required for uploading source maps.
+   */
+  release?: {
+    /**
+     * Unique identifier for the release you want to create.
+     * This value can also be specified via the SENTRY_RELEASE environment variable.
+     *
+     * Defaults to automatically detecting a value for your environment. This includes values for Cordova, Heroku,
+     * AWS CodeBuild, CircleCI, Xcode, and Gradle, and otherwise uses the git HEAD's commit SHA (the latter requires
+     * access to git CLI and for the root directory to be a valid repository).
+     *
+     * If you didn't provide a value and the plugin can't automatically detect one, no release will be created.
+     */
+    name?: string;
+
+    /**
+     * Whether the plugin should inject release information into the build for the SDK to pick it up when
+     * sending events.
+     *
+     * Defaults to `true`.
+     */
+    inject?: boolean;
+  };
+  /**
+   * Options to further customize the Sentry Vite Plugin (@sentry/vite-plugin) behavior directly.
+   * Options specified in this object take precedence over the options specified in
+   * the `sourcemaps` and `release` objects.
+   *
+   * @see https://www.npmjs.com/package/@sentry/vite-plugin/v/2.14.2#options which lists all available options.
+   *
+   * Warning: Options within this object are subject to change at any time.
+   * We DO NOT guarantee semantic versioning for these options, meaning breaking
+   * changes can occur at any time within a major SDK version.
+   *
+   * Furthermore, some options are untested with SvelteKit specifically. Use with caution.
+   */
+  unstable_sentryVitePluginOptions?: Partial<SentryVitePluginOptions>;
+};
+
+type BundleSizeOptimizationOptions = {
+  /**
+   * If set to `true`, the plugin will attempt to tree-shake (remove) any debugging code within the Sentry SDK.
+   * Note that the success of this depends on tree shaking being enabled in your build tooling.
+   *
+   * Setting this option to `true` will disable features like the SDK's `debug` option.
+   */
+  excludeDebugStatements?: boolean;
+
+  /**
+   * If set to true, the plugin will try to tree-shake performance monitoring statements out.
+   * Note that the success of this depends on tree shaking generally being enabled in your build.
+   * Attention: DO NOT enable this when you're using any performance monitoring-related SDK features (e.g. Sentry.startTransaction()).
```

Comment:
l: I guess this is copy pasted from somewhere but I just noticed this:

```suggestion
   * Attention: DO NOT enable this when you're using any performance monitoring-related SDK features (e.g. Sentry.startSpan()).
```